plugins/dell-dock/fu-dell-dock-plugin.c

```diff
@@ -38,72 +67,93 @@ fu_dell_dock_plugin_create_node(FuPlugin *plugin, FuDevice *device, GError **err
 	return TRUE;
 }
 
+static guint8
```

Comment:
shouldn't the return value be the enum type you defined in rsgen not guint8?

Reply:
understood, fixed in [8f86ae8](https://github.com/fwupd/fwupd/commit/8f86ae85077835c77fcf446e55a2632f3d8ac61b) also some other places.